src/ACEtk/details/verify/src/size.hpp

```diff
@@ -0,0 +1,9 @@
+template< typename R1, typename R2 >
+inline bool equalSize( R1& a1, R2& a2 ){
+  if( a1.size() != a2.size() ){
```

Comment:
More meaningful error messages can be made by deferring logging to the client code

Reply:
The intent is for the client code to *also* provide more meaningful messages.